dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/versioning/Navigation.java

```diff
@@ -201,6 +204,12 @@ private Item getItemById() throws SQLException
         if (itemId != null)
         {
             item = itemService.find(this.context, itemId);
+        } else {
+            String itemIDParam = parameters.getParameter("itemID", null);
+            if (StringUtils.isNotBlank(itemIDParam)) {
+                itemId = UUID.fromString(itemIDParam);
+                item = itemService.find(context, itemId);
+            }
```

Comment:
Just another minor formatting/styling nitpick... this has some muddled use of blocks / brackets which makes it a little hard to parse the tests, and doesn't match the DSpace convention of
`if(test) {
    // ...
}
else {
    // ...
}`

(again, I'll change this myself just to save time, wanted to make a note here though so you can see why I'm messing with your PR!)